install.sh

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env bash
+# INSTALL DDB ON WSL WITH AN OFFICIAL UBUNTU VERSION
+
+# Check if the script is run as root if not rerun as root
+if [ "$(id -u)" != "0" ]; then
+   sudo $0
+    exit $?
+fi
+DDB_USER=${DDB_USER:-$SUDO_USER}
+
+if [ -z "$DDB_USER" ]; then
+    echo "DDB_USER is not defined"
+    exit 1
+fi
+
+#Update and upgrade packages
+apt-get update -y
+apt-get upgrade -y
+
+#Install and configuration of Docker
+## Add Docker's official GPG key:
+apt-get update -y
+apt-get install ca-certificates curl make -y
+install -m 0755 -d /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg -o /etc/apt/keyrings/docker.asc
+chmod a+r /etc/apt/keyrings/docker.asc
+
+## Add the repository to Apt sources:
+echo \
+  "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.asc] https://download.docker.com/linux/ubuntu \
+  $(. /etc/os-release && echo "$VERSION_CODENAME") stable" | \
+  tee /etc/apt/sources.list.d/docker.list > /dev/null
+
+apt-get update -y
+
+apt-get install docker-ce docker-ce-cli containerd.io docker-buildx-plugin docker-compose-plugin -y
+
+## By default, the Docker daemon requires root privileges to run. To use Docker without sudo, add your user to the Docker group
+usermod -aG docker $DDB_USER
+
+## Change permissions of the Docker socket
+chmod 666 /var/run/docker.sock
```

Comment:
Not sure we need this one. What is it for ?

Reply:
should be enough to just refresh the user's group with `newgrp docker`
source: https://docs.docker.com/engine/install/linux-postinstall/